my_list.rb

```diff
@@ -0,0 +1,9 @@
+class MyList
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] Great work so far guys! but there seems to be some confusion as you should only create a single class called `MyList` because I noticed you have a class with the same name in the `testing.rb` file so I'd suggest handling the `MyList` class in this file alone and since `MyEnumerable` module should be included in `MyList` class make sure to include the `MyEnumerable` module as well as import the `my_enumerable` file here.